lighthouse-core/test/lib/network-request-test.js

```diff
@@ -71,6 +71,45 @@ describe('NetworkRequest', () => {
     });
   });
 
+  describe('update protocol for Lightrider', () => {
+    function getRequest() {
+      return {
+        protocol: 'http/1.1',
+        responseHeaders: [{name: NetworkRequest.HEADER_PROTOCOL_IS_H2, value: '1'}],
+      };
+    }
+
+    it('does nothing if not Lightrider', () => {
+      const req = getRequest();
+
+      const devtoolsLog = networkRecordsToDevtoolsLog([req]);
+      const record = NetworkRecorder.recordsFromLogs(devtoolsLog)[0];
+
+      expect(record.protocol).toStrictEqual('http/1.1');
+    });
+
+    it('updates protocol if Lightrider', () => {
+      const req = getRequest();
+
+      const devtoolsLog = networkRecordsToDevtoolsLog([req]);
+      global.isLightrider = true;
```

Comment:
add an `afterEach` for `global.isLightrider = undefined`? :)

Reply:
already done in before each